netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js

```diff
@@ -183,6 +191,34 @@ const CandidatesTable = (props) => {
       });
   };
 
+  const sendUserFeedBack = async (id, email, subject, data) => {
+    setOpenDel(false);
+    setLoading(true);
+    const body = {
+      email: email,
+      subject: subject,
+      message: userFeedbackMessage
+    };
+
+    sendUserFeedbackApi(body)
+      .then((res) => {
+        setUserFeedbackMessage('');
+        modifyCandidate(id, data);
+        setLoading(false);
+      })
+      .catch((err) => {
+        setUserFeedbackMessage('');
+        setLoading(false);
+        return dispatch(
+          updateMainAlert({
+            show: true,
+            message: err.response.data.message,
+            severity: 'error'
+          })
+        );
+      });
+  };
+
```

Comment:
Changes: Sends feedback and calls modify candidate

Reply:
Alright, thanks @Codebmk , great stuff!